File: src/cpp/Appointment.cpp
#include <string>

class Appointment {
private:
    std::string m_DoctorName;
    std::string m_AppointmentDate;
    std::string m_TimeSlot;
    bool m_Accepted;

public:
    // Constructor
    Appointment(const std::string& doctorName, const std::string& appointmentDate, const std::string& timeSlot)
        : m_DoctorName(doctorName), m_AppointmentDate(appointmentDate), m_TimeSlot(timeSlot), m_Accepted(false) {}

    // Additional methods can be added if needed
};

File: src/models/Appointment.java
package models;
public class Appointment {
    private String m_DoctorName;
    private String m_AppointmentDate;
    private String m_TimeSlot;
    private boolean m_Accepted;

    public Appointment(String m_DoctorName, String m_AppointmentDate, String m_TimeSlot)
    {
        this.m_AppointmentDate = m_AppointmentDate;
        this.m_DoctorName = m_DoctorName;
        this.m_TimeSlot = m_TimeSlot;
        m_Accepted = false;
    }
}
